clustered/server/src/main/java/org/ehcache/clustered/server/store/ClusterTierActiveEntity.java

```diff
@@ -149,15 +149,18 @@ public void addStateTo(StateDumpCollector dump) {
     {
       Map<ClientDescriptor, ClusterTierClientState> clients = new HashMap<>(connectedClients);
       dump.addState("clientCount", String.valueOf(clients.size()));
-      StateDumpCollector clientsDump = dump.subStateDumpCollector("clients");
-      int idx = 0;
+
+      List<Map> allClients = new ArrayList<>();
```

Comment:
you mean size of this Map right? 
Map<ClientDescriptor, ClusterTierClientState> clients = new HashMap<>(connectedClients);